lib/core/elb_wrapper.rb

```diff
@@ -11,8 +11,13 @@ def elb_client
   
   public
   
-  def elb_arn(region, account_id, elb_name)
-    "arn:aws:elasticloadbalancing:#{region}:#{account_id}:loadbalancer/#{elb_name}"
+  def elb_arn(elb_name)
+    account_id = Aws::IAM::CurrentUser.new.arn.match(/^arn:aws:iam::(\d+)/)[1]
+    "arn:aws:elasticloadbalancing:#{availability_zone}:#{account_id}:loadbalancer/#{elb_name}"
+  end
```

Comment:
@mccalluc this seems like an odd way of getting the ELB's ARN -- although I can't seem to find it anywhere in the AWS web console either. Where did you find the it?

Aside, I tried using `Aws::ElasticLoadBalancer.new.describe_load_balancers` and it's unable to authenticate... so not sure what's going on there. I've set the credentials in `Aws.config` to the values in my `~/.aws/credentials` file, but no dice.